cg/meta/compress/compress.py

```diff
@@ -33,6 +40,17 @@ def set_dry_run(self, dry_run: bool):
         if self.crunchy_api.dry_run is False:
             self.crunchy_api.set_dry_run(dry_run)
 
+    def get_flow_cell_name(self, fastq_path: Path) -> str:
+        """
+        Extract the flow cell name from a fastq Path assuming fastq files are kept in their
+        demultipelxed path and the following run_name convention:
+
+            - <date>_<machine>_<run_numbers>_<A|B><flowcell_id>:
```

Comment:
```suggestion
            - <date>_<machine>_<run_numbers>_<A|B><flow_cell_id>:
```
Here is says flow_cell_id is that the same thing as the flow cell name in this context? If so, I think we should use one consistently.